### PR TITLE
[Teams][Replacedlinkforwebauthentication][1492166]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Replaced link for add authentication to the Teams bot and changes for grammar and language. Please review.

### CODE REVIEW

File: msteams-platform/concepts/authentication/authentication.md [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
 
-You'll need to use the web-based authentication flow for [tabs](~/tabs/what-are-tabs.md), and can choose to use it with [conversational bots](~/bots/what-are-bots.md) or [messaging extensions](~/messaging-extensions/what-are-messaging-extensions.md). You'll use the [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client) in a web content page to enable authentication, then embed that content page in a tab, a configuration page, or a task module. If you want to use the web-based authentication flow with a conversational bot, you'll need to [use a task module with a bot](~/task-modules-and-cards/task-modules/task-modules-bots.md).
+Use the web-based authentication flow for [tabs](~/tabs/what-are-tabs.md) and choose to use it with [conversational bots](~/bots/what-are-bots.md) or [messaging extensions](~/messaging-extensions/what-are-messaging-extensions.md). Use the [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client) in a web content page to enable authentication. Then embed that content page in a tab, a configuration page, or a task module. For more information on web-based authentication flow, see:
```
Comment (KirtiPereira) [inline msteams-platform/concepts/authentication/authentication.md:18]:
```suggestion
Use the web-based authentication flow for [tabs](~/tabs/what-are-tabs.md) and choose to use it with [conversational bots](~/bots/what-are-bots.md) or [messaging extensions](~/messaging-extensions/what-are-messaging-extensions.md). Use the [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client) in a web content page to enable authentication. After enabling authentication, embed the content page in a tab, a configuration page, or a task module. For more information on web-based authentication flow, see:
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@v-rodriguezd - please verify and incorporate the suggestion. Add SME to approve the PR.

File: msteams-platform/concepts/authentication/authentication.md [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
 
-* [Authentication flow in tabs](~/tabs/how-to/authentication/auth-flow-tab.md) describes how tab authentication works in Teams. This shows a typical web based authentication flow used for tabs.
-* [Azure AD authentication in tabs](~/tabs/how-to/authentication/auth-tab-AAD.md) describes how to connect to Azure Active Directory from within a tab in your app in Teams.
-* [Silent authentication (Azure AD)](~/tabs/how-to/authentication/auth-silent-AAD.md) describes how to reduce sign-in/consent prompts in your app using Azure Active Directory.
-* Web-based authentication in [dotnet/C#](https://github.com/OfficeDev/microsoft-teams-sample-complete-csharp) or [JavaScript/Node.js](https://github.com/OfficeDev/microsoft-teams-sample-complete-node)
+* [Add authentication to the Teams bot](~/bots/how-to/authentication/add-authentication.md) describes how to use web-based authentication flow with a conversational bot.
```
Comment (Trinetra-MSFT) [inline msteams-platform/concepts/authentication/authentication.md:20]: All links are leading to 404 not found, Update to point correct links

Comment (v-rodriguezd) [reply msteams-platform/concepts/authentication/authentication.md:20]: The links are working. I haven't changed any link as they are working in master the links if not working should work after merging. Please check.

**Review by Trinetra-MSFT — APPROVED** [review]
Good to go

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved the PR.

**Pull request merged**